### ISSUE TITLE
Incorrect link on topic titles on cards when nodebb running from subfolder

### ISSUE THREAD

**Issue by joolswills** [NodeBB-Community/nodebb-plugin-recent-cards#50]
Running recent cards 2.0.11

nodebb 1.15.x

You can see the issue here https://retropie.org.uk/forum/

The /forum is not included.

**Comment by joolswills** [comment]
I fixed it by making this change

```
--- static/templates/partials/nodebb-plugin-recent-cards/header.tpl.old	2020-11-05 23:22:42.498297380 +0000
+++ static/templates/partials/nodebb-plugin-recent-cards/header.tpl	2020-11-05 23:19:20.489360929 +0000
@@ -10,7 +10,7 @@
 			<div class="recent-card">
 				<div class="recent-card-body">
 					<div>
-						<h4><a href="{config.relative}/topic/{topics.slug}{{{ if topics.bookmark }}}/{topics.bookmark}{{{ end }}}">{topics.title}</a></h4>
+						<h4><a href="{config.relative_path}/topic/{topics.slug}{{{ if topics.bookmark }}}/{topics.bookmark}{{{ end }}}">{topics.title}</a></h4>
 					</div>
 					<div class="clearfix">
 						<div class="pull-left">
```

**Comment by barisusakli** [comment]
@joolswills thanks for reporting, fixed in 2.0.12
